eks/private_subnet.tf

```diff
@@ -14,10 +14,15 @@ resource "aws_subnet" "private" {
   }
 }
 
+resource "aws_eip" "eips" {
```

Comment:
Maybe you want to do this only if var.eips is 0

Reply:
changed the default and checks the length